metron-interface/metron-alerts/.nvmrc

```diff
@@ -1 +1 @@
-v9.11.1
+v10.15.1
```

Comment:
With .nvmrc we can pin down the node/npm version both on Travis and local.
Upgraded to the latest LTS node version.